STEERING-COMMITTEE.md

```diff
@@ -263,12 +264,13 @@ In order to transition from the original, organization-appointed Steering
 Committee structure (the "bootstrap" committee), elections will be phased in
 over 14 months.
 
-In October 2020, four positions will be elected, including three Contributing
-seats and one End-User seat. One Google, one IBM/RedHat, and one VMware seat
-will be retained from the current Steering Committee.
+In October 2020, two positions will be elected, including two Contributing
```

Comment:
I think we should remove or annotate the language on L210 around end user seats, or remove the language entirely and just say they are planned but not effective yet.

Reply:
Rewrote the section around 210 to remove the mechanics pending further refinement of the eligibility criteria.